### PR TITLE
Shrink size of Extensibility.Standalone image

### PR DESCRIPTION
**Pull request by rynowak**
This change switches the extensibility.standalone image to use alpine
and self-contained publishing.

This has the overall effect of reducing size from 250mb -> 112mb.

I did not turn on assembly trimming because there are many many warnings
showing up from libraries.

### CODE REVIEW

**Comment by rynowak** [comment]
@shenglol - are you using this project for anything? We're using this as an image (built from this repo) and really need to reduce the size to reduce our install time.

File: src/Extensibility.Standalone/CrudHelper.cs [hunk 1]
```diff
@@ -1,14 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Extensibility.Core.Contract;
+using Extensibility.Core.Data;
```
Comment (rynowak) [inline src/Extensibility.Standalone/CrudHelper.cs:8]: Warnings as errors prevents this from building without fixing the style errors.

**Comment by shenglol** [comment]
> 

No, I'm not using this for anything. But I think we will deprecate the old `Extensibility.*` projects once the [new dev host](https://github.com/Azure/bicep-extensibility/tree/main/src/Azure.Deployments.Extensibility.DevHost) is picked up by ARM, which is expected to happen in two weeks.

**Comment by rynowak** [comment]
> > @shenglol - are you using this project for anything? We're using this as an image (built from this repo) and really need to reduce the size to reduce our install time.
> 
> No, I'm not using this for anything. But I think we will deprecate the old `Extensibility.*` projects once the [new dev host](https://github.com/Azure/bicep-extensibility/tree/main/src/Azure.Deployments.Extensibility.DevHost) is picked up by ARM, which is expected to happen in two weeks.

Sounds good. We're wrapping up a release right now, so early in the next milestone is a good time for us to get up to date. /cc @jkotalik  as FYI

**Pull request merged**